packages/regex-common/.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+	"extends": ["../../.eslintrc.json"],
+	"ignorePatterns": ["!**/*"],
+	"overrides": [
```

Comment:
We moeten wel eens kijken of die overrides wel nodig zijn. Want uiteindelijk, als we niks gaan overschrijven en we willen dat ook liever niet, dan is het beter om dit gewoon weg te nemen.

Reply:
Ik heb het toegevoegd aan de algemene todo's in het canvas van het Slack kanaal.